misk-hibernate-testing/src/main/kotlin/misk/jdbc/VitessScaleSafetyChecks.kt

```diff
@@ -305,10 +305,10 @@ class VitessScaleSafetyChecks(
     val request = Request.Builder()
         .url("http://localhost:27000/debug/vars")
         .build()
+    val response = okHttpClient.newCall(request).execute()
```

Comment:
this was part of the original commit to fix flaky tests, that didn't need to be reverted.